modules/exploits/multi/http/php_useragentt_rce.rb

```diff
@@ -0,0 +1,124 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+class MetasploitModule < Msf::Exploit::Remote
+  Rank = ExcellentRanking
+
+  include Msf::Exploit::Remote::HttpClient
+
```

Comment:
Does it make sense to use yjr command stager mixin here? 🤔 
https://github.com/rapid7/metasploit-framework/wiki/How-to-use-command-stagers

Reply:
I don't know much about this mixin but will give it a try, might be helpful in case of windows targets. :)